internal/querycoordv2/server.go

```diff
@@ -735,12 +735,11 @@ func (s *Server) tryHandleNodeUp() {
 	log := log.Ctx(s.ctx).WithRateGroup("qcv2.Server", 1, 60)
 	ctx, cancel := context.WithTimeout(s.ctx, Params.QueryCoordCfg.CheckHealthRPCTimeout.GetAsDuration(time.Millisecond))
 	defer cancel()
-	reasons, err := s.checkNodeHealth(ctx)
+	err := s.checkNodeHealth(ctx)
```

Comment:
there is no reason to block new node up because of old node is not healthy.
@weiliu1031 please double confirm